Gemfile

```diff
@@ -3,7 +3,7 @@ source "https://rubygems.org"
 gemspec
 
 gem "rails", "~> 6.0.0"
-ENV.fetch("ALCHEMY_BRANCH", "6.0-stable").tap do |branch|
+ENV.fetch("ALCHEMY_BRANCH", "6.1-stable").tap do |branch|
```

Comment:
This change should be extracted into a separate PR that provides Alchemy 6.1 compatibility 

In that PR we also want to make sure to test both 6.0 and 6.1